spdx-utils/src/test/kotlin/SpdxDeclaredLicenseMappingTest.kt

```diff
@@ -41,19 +41,28 @@ class SpdxDeclaredLicenseMappingTest : WordSpec({
     }
 
     "The mapping" should {
-        "contain only unparsable keys" {
-            val parsableLicenses = SpdxDeclaredLicenseMapping.mapping.filter { (declaredLicense, _) ->
+        "not contain single ID strings" {
+            val minTokens = 2
+
+            val licenseIdMapping = SpdxDeclaredLicenseMapping.mapping.filter { (_, expression) ->
+                expression is SpdxLicenseIdExpression
+            }
+
+            licenseIdMapping.keys.forAll { declaredLicense ->
                 try {
-                    // Restrict parsing to SPDX license identifier strings as otherwise almost anything could be parsed,
-                    // but we do want to have mappings e.g. for something like "CDDL or GPLv2 with exceptions".
-                    SpdxExpression.parse(declaredLicense, Strictness.ALLOW_DEPRECATED)
-                    true
+                    val tokens = getTokensByTypeForExpression(declaredLicense)
+                    val condition = tokens.size >= minTokens
+
+                    "It is $condition that '$declaredLicense' has at least $minTokens tokens" shouldBe
```

Comment:
Hm, with the `forAll` proposal I was mainly up for eliminating this error message work around as I find it quite hard to grasp compared to a normal assertion, probably as it is not a `standard` solution. Do you see it similar and maybe have an idea how to get rid of it?

I thought my `shouldBeGreaterOrEqual` proposal of my previously pasted code snip would work out?

Reply:
Yes, I simply forgot to change that code, too. Done now.

Reply:
Thanks!